package.json

```diff
@@ -81,7 +81,7 @@
     "moment-duration-format": "2.2.2",
     "patternfly": "3.53.1",
     "patternfly-bootstrap-combobox": "1.1.7",
-    "patternfly-react": "2.11.1",
+    "patternfly-react": "2.17.4",
```

Comment:
I think we should upgrade to the latest available as the risk to upgrade from 2.11.1 to 2.17.4 requires the same amount of regression testing (and potential fixes) as the upgrade from 2.11.1 to 2.29.15.

Also, which ever version gets selected, `patternfly` should be updated to the version used in `patternfly-react`.

Reply:
agree